api.js

```diff
@@ -151,7 +151,7 @@ Api.prototype._run = function (files, _options) {
 					// continue to run.
 					runStatus.handleExceptions({
 						exception: err,
-						file: file
+						file: path.relative('.', file)
```

Comment:
Sure, but that should be done as a separate task. `fork` emits more of these files so I figured I'd make it consistent with that first.